openlibrary/plugins/openlibrary/js/patron-metadata/index.js

```diff
@@ -30,11 +30,11 @@ export function initPatronMetadata() {
 
             $form.append(`
               <details class="aspect-section">
-                <summary>${aspect.label}</summary>
-                <div id="${aspect.label}-question">
-                    <h3>${aspect.description}</h3>
+                <summary><h3>${aspect.label}</h3></summary>
+                <fieldset id="${aspect.label}-question">
+                    <legend>${aspect.description}</legend>
                     ${$choices.prop('outerHTML')}
```

Comment:
Note to self: not related to this change, but not sure why this line is using prop outerHTML and not outerHTML directly...